third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -2350,6 +2355,46 @@ static void fromJsEndReplayCode(
   recordreplay::ExitReplayCode();
 }
 
+// Connects replay globals that must be accessible from all contexts; MUST be called
+// with AutoMarkReplayCode.
+static void ConnectPreambleGlobals(LocalFrame* frame, v8::Isolate *isolate) {
```

Comment:
Ehh, I waffled on this, due to recursion (it just looks distasteful/re-entrant to me); once we fix our context handling, I think we'll end up with a set of Conexts we can just safely iterate over, rather than this traversal.